README.md

```diff
@@ -1,44 +1,64 @@
-The page will reload if you make edits.<br />
-You will also see any lint errors in the console.
-
-### `npm test`
-
-Launches the test runner in the interactive watch mode.<br />
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
-
-### `npm run build`
-
-Builds the app for production to the `build` folder.<br />
-It correctly bundles React in production mode and optimizes the build for the best performance.
-
-The build is minified and the filenames include the hashes.<br />
-Your app is ready to be deployed!
-
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
-
-### `npm run eject`
-
-**Note: this is a one-way operation. Once you `eject`, you can’t go back!**
-
-If you aren’t satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
-
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you’re on your own.
-
-You don’t have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn’t feel obligated to use this feature. However we understand that this tool wouldn’t be useful if you couldn’t customize it when you are ready for it.
-
-## Learn More
-
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
-
-To learn React, check out the [React documentation](https://reactjs.org/).
+# React Date Range Picker
+react-daterange-picker can be used to pick a date range in react and specify a data refresh interval.
+## Demo
+![media](https://github.com/iamPres/react-daterange-picker/blob/new/media/readme-body-1.PNG)
+![media](https://github.com/iamPres/react-daterange-picker/blob/new/media/readme-body-2.PNG)
+![media](https://github.com/iamPres/react-daterange-picker/blob/new/media/readme-timer-1.PNG)
+## Dependencies
+ - install the [package.json](https://github.com/iamPres/react-daterange-picker/blob/master/package.json) dependencies with npm
+
+## Example Usage
+```
+import React from "react";
+import ReactDOM from "react-dom";
+import { MuiThemeProvider, createMuiTheme } from "@material-ui/core/styles";
+import purple from "@material-ui/core/colors/purple";
+import green from "@material-ui/core/colors/green";
+import DateRangePicker from "../src/index.tsx";
+
+const app = document.getElementById("app");
+
+function reset() {
+  console.log("Reset!");
+}
+
+function getData(data) {
+  console.log(data);
+}
+
+ReactDOM.render(
+  <DateRangePicker
+    resetFn={reset}
+    getData={getData}
+    dateFormatter={
+      new Intl.DateTimeFormat("en", {
```

Comment:
Don't include this in the default example, put it in the documentation where you document what the various options are.